### PR TITLE
added TC-072, TC-073, TC-078, TC-079

### PR DESCRIPTION
**Pull request by oksanakirieieva**

### CODE REVIEW

File: test/extended/age.js [hunk 1]
```diff
@@ -46,6 +46,35 @@ describe('Age field suit', function () {
...
         });
 
+        it('TC-072 Age field spin click up works when 1 digit in age input field" ', function () {
+            $(sel.age).setValue(age.one);
+            $(sel.spinUp).click();
+            let input = $(sel.age).getValue();
+            expect(input).toEqual(age.two);
+        });
+
+        it.only('TC-073 Age field spin click down works when digit 2 in age input field" ', function () {
```
Comment (Slyvochka) [inline test/extended/age.js:56]: "it.only" -> it's really means something or you added it by mistake

File: test/extended/age.js [hunk 2]
```diff
@@ -46,6 +46,35 @@ describe('Age field suit', function () {
...
+            $(sel.age).setValue(age.two);
+            $(sel.spinDown).click();
+            let input = $(sel.age).getValue();
+            expect(input).toEqual(age.one);
+        });
+
+        it.only('TC-078 The Age field required', function () {
```
Comment (ivansh83) [inline test/extended/age.js:63]: Yes, please remove .only here and above. You probably left it there by mistake when running individual cases.

**Review by ivansh83 — CHANGES_REQUESTED** [review]

**Review by o-akhadova — COMMENTED** [review]
Everything works, good job! But you should delete 'only' before commit and push

**Review by Slyvochka — APPROVED** [review]
deleted it.only

**Comment by oksanakirieieva** [comment]
deleted, thank you, please check if everything is correct and approve then

**Review by ivansh83 — APPROVED** [review]

**Pull request merged**